doc/api/vm.md

```diff
@@ -662,7 +662,7 @@ defined in the ECMAScript specification.
     when `import()` is called. If this option is not specified, calls to
     `import()` will reject with [`ERR_VM_DYNAMIC_IMPORT_CALLBACK_MISSING`][].
     * `specifier` {string} specifier passed to `import()`
-    * `module` {vm.Module}
```

Comment:
this one should not be changed

Reply:
good catch, went a bit quick with my search and replace 👍